### PR TITLE
AAE-8264: migrate from Travis to GH actions

### PR DESCRIPTION
**Pull request by atchertchian**
Depends on https://github.com/Alfresco/alfresco-build-tools/pull/101

### CODE REVIEW

File: .github/actions/helm-dryrun/action.yml
Comment (atchertchian) [inline .github/actions/helm-dryrun/action.yml]: @gionn wdyt about this action in alfresco-build-tools please? I took it from here: https://github.com/Activiti/activiti-cloud-full-chart/tree/master/.github/actions/helm-dryrun

File: .github/workflows/main.yml
Comment (atchertchian) [inline .github/workflows/main.yml]: @gionn is there a cluster name I can use on Alfresco repositories for testing, please?

Comment (gionn) [reply .github/workflows/main.yml]: you can spin up a new env on the pipeline and then use that here

File: .github/actions/helm-dryrun/action.yml
Comment (gionn) [reply .github/actions/helm-dryrun/action.yml]: not a problem to move there but https://github.com/Activiti/activiti-cloud-full-chart/blob/master/.github/actions/helm-dryrun/action.yml#L30 looks something specific to your repos?

File: .github/workflows/main.yml
Comment (atchertchian) [reply .github/workflows/main.yml]: ok, I think I'll do that a bit later, when the PR is ready for review :+1: 
(the activiti-test one seems to be ok to validate changes for now)

File: .github/actions/helm-dryrun/action.yml
Comment (atchertchian) [reply .github/actions/helm-dryrun/action.yml]: I'm not familiar with the logic enough TBH. I'll duplicate this action in a third repo for now and will check this with Elias before merging, thanks

Comment (atchertchian) [reply .github/actions/helm-dryrun/action.yml]: I got what you were saying reworking the code :+1: 
I'll extract this the common logic to alfresco-build-tools, and will wait for Elias to merge.

File: .github/workflows/main.yml
Comment (gionn) [inline .github/workflows/main.yml]: cancel in progress with a workflow that relies on external resources (a k8s cluster for testing) is usually a bad idea - it can skip the cleanup of resources

Comment (atchertchian) [reply .github/workflows/main.yml]: Indeed, I was looking at ways of cleaning up on cancellation, but that seems to be available only on javascript actions (https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runspost)

Comment (gionn) [reply .github/workflows/main.yml]: there is a specific behaviour when a cancel get triggered, see https://docs.github.com/en/actions/managing-workflow-runs/canceling-a-workflow#steps-github-takes-to-cancel-a-workflow-run

Comment (atchertchian) [reply .github/workflows/main.yml]: I removed the concurrency configuration since the checks are quick

File: .github/workflows/main.yml
Comment (atchertchian) [reply .github/workflows/main.yml]: since this cluster was used by Travis, will keep on using it with GH actions if that's alright

File: .github/actions/helm-dryrun/action.yml
Comment (atchertchian) [reply .github/actions/helm-dryrun/action.yml]: The action has been extracted (see https://github.com/Alfresco/alfresco-build-tools/pull/164) : will review Activiti workflows once that works on alfresco repositories

**Review by igdianov — APPROVED** [review]

**Pull request merged**
